twake/backend/node/src/services/channels/web/schemas.ts

```diff
@@ -42,6 +42,7 @@ const channelSchema = {
     user_member: {},
     direct_channel_members: { type: "array" },
     stats: {},
+    is_readonly: { type: "boolean" },
```

Comment:
For this to work you must add `is_readonly: isWorkspaceAdmin || isChannelOwner` to `twake/backend/node/src/services/channels/services/channel/service.ts` line 143.